pkg/workload/tpch/generate.go

```diff
@@ -213,7 +213,7 @@ func (w *tpch) tpchPartSuppInitialRowBatch(
 		// PS_AVAILQTY random value [1 .. 9,999].
 		availQtyCol[i] = int16(randInt(rng, 1, 9999))
 		// PS_SUPPLYCOST random value [1.00 .. 1,000.00].
-		supplyCostCol[i] = float64(randFloat(rng, 1, 1000, 100))
```

Comment:
This seems out of sync with the comment - should the comment be 10,000 or should the value stay 1,000? Or does this function do something odd?

Reply:
The comment is correct. `randFloat` does do something a bit odd:
```
func randFloat(rng *rand.Rand, x, y, shift int) float32 {
	return float32(randInt(rng, x, y)) / float32(shift)
}
```

Reply:
Just realized that I needed to change the first value too -- done.

Reply:
👍